src/datanode/src/heartbeat.rs

```diff
@@ -241,6 +263,26 @@ impl HeartbeatTask {
         Ok(())
     }
 
+    async fn load_stats(region_server: &RegionServer) -> (u64, Vec<RegionStat>) {
+        let region_ids = region_server.opened_region_ids();
+        let region_stats = region_ids
+            .into_iter()
+            .map(|region_id| RegionStat {
+                // TODO: scratch more info
+                region_id: region_id.as_u64(),
+                table_ident: Some(TableIdent {
+                    table_id: region_id.table_id(),
+                    table_name: None,
+                    engine: "MitoEngine".to_string(),
```

Comment:
Let's redesign the heartbeat related features in the Metasrv first, so that region server's heartbeats don't break too much things. Starting from distinguishing this `engine` field first. I can do that. WDYT @fengjiachun

Reply:
I am working on this